.gitignore

```diff
@@ -6,13 +6,26 @@
 
 # Ignore bundler config.
 /.bundle
+capybara-*.html
+.rspec
+/db/*.pg
+/spec/tmp
+/vendor/bundle
 
 # Ignore all logfiles and tempfiles.
 /log/*
 /tmp/*
 !/log/.keep
 !/tmp/.keep
 
+# TODO Comment out this rule if you are OK with secrets being uploaded to the repo
+config/initializers/secret_token.rb
+config/master.key
+
+# dotenv
+# TODO Comment out this rule if environment variables can be committed
+.env
```

Comment:
I think we need to add .env.sample

Reply:
So. why do we need .env.sample?)

Reply:
Cause you add .env to gitignore and no one know what there should be

Reply:
And how we will load dotenv files? Is there any gems in Gemfile for that?